btk/sampling_functions.py

```diff
@@ -259,6 +259,82 @@ def __call__(self, table, **kwargs):
         return blend_table
 
 
+class DefaultSamplingShear(SamplingFunction):
```

Comment:
Right now this function is basically the same as `DefaultSampling`, I guess it's ok since we want to keep the base one simple for tutorial purposes. 

Another option is to have this new class be a children of DefaultSampling and rewrite the `__call__` to first call the parent class function and then add the shear stuff afterwards. What do you think @thuiop ?

Reply:
I guess this could be a good idea, yes.